modules/url.py

```diff
@@ -73,7 +74,21 @@
 INVALID_WEBSITE = 0x01
 HTML_ENTITIES = { 'apos': "'" }
 
-yt_catch = re.compile('http[s]*:\/\/[w\.]*(youtube.com/watch\S*v=|youtu.be/)([\w-]+)')
+yt_catch = re.compile(r'http[s]*:\/\/[w\.]*(youtube.com/watch\S*v=|youtu.be/)([\w-]+)')
+arxiv_catch = re.compile(
+    r"""http[s]?://                       # durr
+        [^/]*                             # for fr./www./&c.
+        (xxx\.lanl\.gov|arxiv\.org)/      # xxx.lanl.gov still works
+        [a-z]+/                           # for the category
+        (\d{4}\.\d{4,5}|[a-z\-\.]+/\d{7}) # arXiv id in group(2)""", re.X)
+
+def setup(jenni):
+    # enable show_title_auto by default, but disable
+    # if show_title_auto=False in config
+    if (not hasattr(jenni.config, 'show_title_auto') or 
+            jenni.config.show_title_auto is True):
+        show_title_auto.rule = '(?iu).*(%s?(http|https)(://\S+)).*' % (EXCLUSION_CHAR)
+        show_title_auto.priority = 'high'
```

Comment:
@myano @kaneda yano's original idea was to check jenni.config.show_title_auto within show_title_auto(), but that means you're triggering on a very general regex for no reason if it is set to False